common/parser.hpp

```diff
@@ -139,8 +139,12 @@ struct parameter {
 };
 
 struct section {
-    std::string name, title, format_type, data;
-    int offset, size;
+    std::string name = "";
```

Comment:
Really, this is an issue?

Reply:
Well, it really is the function of how it's used I suppose.
I choose to disregard any uninitialised variables warnings if I see they are later assigned to, however in this case, section fields are assigned only if the XML has the corresponding attributes and section is stack allocated, so at least offset and size have the potential to be garbage. I am not sure about the string, so I initialised it just in case

Reply:
I was asking about string, it has a default c'tor , looks like a false positive if they meant it and not only the offset and size